### ISSUE TITLE
function validip doesn't look properly in the API docs

### ISSUE THREAD

**Issue by curiousrohan** [webpy/webpy#403]
This is how it looks.

![screenshot from 2017-02-18 15-08-17](https://cloud.githubusercontent.com/assets/20515859/23092230/31b199ea-f5ec-11e6-8bc6-3428ae6fe9b3.png)

**Comment by anandology** [comment]
Could you please generate the docs and see if the issue is still there? You can generate the API docs by running the following from the root of webpy repository.

    PYTHONPATH=. python tools/makedoc.py >/dev/null

**Comment by curiousrohan** [comment]
I'm getting this error

```
[rohan@localhost webpy]$ PYTHONPATH=. python tools/makedoc.py >/dev/null
Traceback (most recent call last):
  File "tools/makedoc.py", line 172, in <module>
    main(sys.argv[1:])
  File "tools/makedoc.py", line 167, in main
    mod = __import__(name, {}, {}, 'x')
  File "../web/webopenid.py", line 26, in <module>
    import openid.consumer.consumer
ImportError: No module named openid.consumer.consumer
```

**Comment by anandology** [comment]
Can you see if you can ignore the modules creating trouble and continue generating the API docs?

**Comment by anandology** [comment]
Fixed in #406 by @curiousrohan. The docs will be fixed in the next release when the API docs are regenerated.
